integrations/datadog/overrides.py

```diff
@@ -33,12 +33,29 @@ class SLOHistoryResourceConfig(ResourceConfig):
     selector: SLOHistorySelector
 
 
-class DataDogPortAppConfig(PortAppConfig):
-    resources: list[SLOHistoryResourceConfig | ResourceConfig] = Field(
-        default_factory=list
+class DatadogSelector(BaseModel):
+    metric: str = Field(alias="metric", required=True)
+    env: str = Field(alias="env", default="*")
```

Comment:
Default prod

Reply:
How about if another user called it `production`?

Reply:
I thought its auto defined by datadog 🤔

Reply:
@Tankilevitch, what is reliable is that we can always find the tag env but when setting up the datadog agent on a host, a user specifies the value of the tag. I setup host and specified prod and staging. But another user could set production, test or something else